src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedCSSData = {};
+  const splitedSource = sourceString.split(';');
+
+  splitedSource
+    .map((item) => {
+      const formatedPropAndValueObject = item
+        .split(':')
+        .map((elem) => elem.trim());
+
+      return formatedPropAndValueObject;
```

Comment:
The variable 'formatedPropAndValueObject' is redundant. You can destructure the property and value directly in the forEach callback without creating this intermediate variable.